Doc/library/pdb.rst

```diff
@@ -379,6 +379,13 @@ can be overridden by the local file.
    Execute the current line, stop at the first possible occasion (either in a
    function that is called or on the next line in the current function).
 
+.. pdbcommand:: si | stepi
+
+   Execute the current instruction, stop at the first possible occasion (either in a
```

Comment:
```suggestion
   Execute the current bytecode instruction, stop at the first possible occasion (either in a
```

I guess it might be unclear what "instruction" mean. The same applies to other mentions of "instruction".